debian/buster/Dockerfile

```diff
@@ -48,7 +48,7 @@ COPY bootstrap.sh /
 # port provision
 EXPOSE 3310
 
-RUN chown clamav:clamav bootstrap.sh check.sh envconfig.sh /etc/clamav/clamd.conf /etc/clamav/freshclam.conf && \
+RUN chown clamav:clamav bootstrap.sh check.sh envconfig.sh /etc/clamav /etc/clamav/clamd.conf /etc/clamav/freshclam.conf && \
```

Comment:
Why did you remove /etc/clamav as folder from chown?

Reply:
I added /etc/clamav. It was owned by root previously and creating the backup config file and the sed backups were failing, so you couldn't use config variables or provide your own config file.